src/types.ts

```diff
@@ -16,6 +16,11 @@ export interface Video {
 
 export interface Media {
   screenshots: Image[];
+  covers: {
```

Comment:
```suggestion
  covers: Record<string, { url: string }>,
```
Мы же вроде так пишем